src/3rdparty/walkontable/src/border.js

```diff
@@ -441,18 +448,72 @@ class Border {
     this.leftStyle.height = `${height}px`;
     this.leftStyle.display = 'block';
 
-    const delta = Math.floor(this.settings.border.width / 2);
+    let delta = Math.floor(this.settings.border.width / 2);
+
+    if (delta > 1) {
+      delta += delta;
+    }
+
+    let extra = 0;
 
-    this.bottomStyle.top = `${top + height - delta}px`;
+    if (this.settings.border.width > 3) {
+      extra = (this.settings.border.width % delta) - 1;
+    } else if (this.settings.border.width === 3) {
+      extra = 1;
+    }
+
+    this.bottomStyle.top = `${top + height - delta - extra}px`;
     this.bottomStyle.left = `${left}px`;
     this.bottomStyle.width = `${width}px`;
     this.bottomStyle.display = 'block';
 
     this.rightStyle.top = `${top}px`;
-    this.rightStyle.left = `${left + width - delta}px`;
+    this.rightStyle.left = `${left + width - delta - extra}px`;
     this.rightStyle.height = `${height + 1}px`;
     this.rightStyle.display = 'block';
 
+    if (this.settings.right && this.settings.right.hide !== true) {
```

Comment:
I'm not sure this delta calculations works properly. If I use `width: 10px` then borders looks strange.
![screenshot 2019-02-14 at 10 45 09](https://user-images.githubusercontent.com/11849621/52778278-f56dc800-3045-11e9-8208-c69d5d93a3ff.png)

Reply:
In pro version is ok.